<launch>
  <arg name="camera_id" default="/"/>
  <arg name="camera_info_src" default="/camera_info"/>
  <arg name="use_path_info" default="false"/> <!-- USE VectorMap Server to publish only TrafficSignals on current lane-->

  <node pkg="feat_proj_lanelet2_v2" type="feat_proj_lanelet2_v2" name="feature_proj_lanelet2_v2" output="log">
    <remap from="camera_info" to="$(arg camera_id)$(arg camera_info_src)"/>
    <param name="camera_frame" type="str" value="front_camera_optical_link"/>
    <param name="use_path_info" type="bool" value="$(arg use_path_info)"/>
  </node>
</launch>
